src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/BlockCollection.js

```diff
@@ -128,8 +129,8 @@ class BlockCollection<T: string, U: {type: T}> extends React.Component<Props<T,
         return indexes;
     }
 
-    @action handleAddBlock = (insertionIndex: number) => {
-        const {defaultType, onChange, value} = this.props;
+    @action handleAddBlock = async(insertionIndex: number) => {
```

Comment:
isnt that a keyword? then should be a space here not?

```suggestion
    @action handleAddBlock = async (insertionIndex: number) => {
```